awscli/customizations/configure/set.py

```diff
@@ -105,3 +105,4 @@ def _run_main(self, args, parsed_globals):
             section = profile_to_section(profile)
         updated_config = {'__section__': section, varname: value}
         self._config_writer.update_config(updated_config, config_filename)
+        return 0
```

Comment:
The new test class `test_configure` inherits from expects a return code for ALL commands.